docs/install.md

```diff
@@ -7,12 +7,16 @@
 
 ## Requirements
 
-- [Kubernetes](https://kubernetes.io/) v1.9 or later
+- [Kubernetes](https://kubernetes.io/) v1.9 or later with an SSD storage class
+  - Cluster role administrator access
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) v1.9.7 or later
 - Access to server infrastructure on which you can create a Kubernetes cluster (see
   [resource allocation guidelines](scale.md)).
 - [Sourcegraph Enterprise license](./configure.md#add-license-key). You can run through these instructions without one, but you must obtain a license for instances of more than 10 users.
-
+- A domain name for your Sourcegraph instance ([to enable SSL/TLS](https://github.com/sourcegraph/deploy-sourcegraph/blob/master/docs/configure.md#configure-tlsssl))
+- A certificate (self-signed or from a service such as [Let's Encrypt](https://letsencrypt.org/))
```

Comment:
There are a number of options here:

1) A valid certificate issued by a trusted certificate authority (such as Comodo, RapidSSL, etc.)
2) A self-signed certificate that you can distribute and install across all of your user's machines
3) An existing reverse proxy that provides SSL termination for the (such as Cloudflare)

Reply:
What's a good way to describe this at a high level? E.g. "A way to ___" or "A method for ___" or "A certificate or ___". 

I.e., what's the header bullet if this list were sub-bullets?

Reply:
i would say: "A TLS certificate"